addons/survey/models/survey_user_input.py

```diff
@@ -623,6 +625,24 @@ def _message_get_suggested_recipients(self):
                 )
         return recipients
 
+    def _notify_new_participation_subscribers(self):
+        partner_root_id = self.env.ref('base.partner_root').id
+        subtype_id = self.env.ref('survey.mt_survey_survey_user_input_completed', raise_if_not_found=False)
+        if not self.ids or not subtype_id:
+            return
+        recipients_data = self.env['mail.followers']._get_recipient_data(self.survey_id, 'notification', subtype_id.id)
+        followed_survey_ids = [survey_id for survey_id, followers in recipients_data.items() if followers]
+        for user_input in self.filtered(lambda user_input: user_input.survey_id.id in followed_survey_ids):
+            user_input.message_post(
```

Comment:
Then maybe you can already filter out people not belonging to 'group_survey_user' ... otherwise you might be upset receiving 2K notifications for records you can't access.

Reply:
Mainly because we don't know what people will do with followers and subtypes, but I think we don't want other people to receive notifications like "Raoul participated to Burger Quiz", if they can't access the user input not comment anything ^^ .

Reply:
Normally they shouldn't be allowed to subscribe to the subtype in that case. 
What I mean is that the just added complexity here would prevent creating the message *only* if *all* subscribers are actually not in group_survey_users... But why not, heh. 
Some way of filtering recipients based on a group could be nice indeed, but I didn't find it.

Reply:
"Normally they shouldn't be allowed to subscribe to the subtype in that case."
-> what prevent people from doing so ?

Reply:
`self.check_access_rights('read')` in `message_subscribe`, which can be avoided, is that what you mean?
Still, what we did will not prevent those hackers from being notified in that case...

Reply:
Not sure what we are discussing actually. My thought is that you may subscribe people to a record (aka adding people as followers, independently from their access), which does not mean they should receive notifications that make no sense for them.